neo/Network/P2P/EmptyActor.cs

```diff
@@ -0,0 +1,16 @@
+using Akka.Actor;
+
+namespace Neo.Network.P2P
+{
+    class EmptyActor : UntypedActor
```

Comment:
Please use `ActorRefs.NoSender`.

Reply:
fixed

Reply:
It'll always throw exceptions when use `ActorRefs.NoSender`.

![image](https://user-images.githubusercontent.com/43396869/69407527-a032b580-0d3f-11ea-9384-6a5f560118c7.png)

The solution is to use `ActorRefs.NoBody`.